types/child-process-promise/index.d.ts

```diff
@@ -0,0 +1,107 @@
+// Type definitions for child-process-promise 2.2
+// Project: https://github.com/patrick-steele-idem/child-process-promise
+// Definitions by: Luis Paulo <https://github.com/TheDSCPL>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+/// <reference types="node" />
+
+import {
+    ChildProcess,
+    ExecFileOptionsWithBufferEncoding, ExecFileOptionsWithOtherEncoding,
+    ExecFileOptionsWithStringEncoding,
+    ExecOptions,
+    ForkOptions,
+    SpawnOptions
+} from 'child_process';
+
+/**
+ * Simple wrapper around the child_process module that makes use of promises
+ */
+
+// stop exporting everything by default
+export {};
```

Comment:
Don't need this. Please remove it

Reply:
I was isolating implementation details because I didn't think named exports were needed for those return types because the important part comes from the 'child_process' package. Nevertheless, I've removed the `export {};`, its comment and exported the interfaces that were being so.